js/Marquee.js

```diff
@@ -0,0 +1,29 @@
+import {fetchData} from "./utils.js"
+
+export class Marquee {
+    constructor (element){
+       this.element = element;
+    }
+    
+    async load(){
+      console.log("shalom!");
+      const spinner = document.createElement("div");
+      spinner.id = "loading";
+      spinner.className = "spinner-border d-none";
+      spinner.role = "status";
+      this.element.appendChild(spinner);
```

Comment:
All of those can be written with one command:
this.element.innerHTML += `<div id="loading" class="spinner-border d-none" role="status"> </div>`